.github/workflows/test.yml

```diff
@@ -456,12 +456,16 @@ jobs:
         fuzz-seconds: 300
         dry-run: false
         language: go
+    - name: Set artifacts_path in env (workaround for actions/upload-artifact#176)
+      if: steps.run.outcome != 'success' && steps.build.outcome == 'success'
+      run: |
+        echo "artifacts_path=$(realpath .)" >> $GITHUB_ENV
     - name: upload crash
-      uses: actions/upload-artifact@v3
+      uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # v4.4.0
```

Comment:
Other breaking changes in this version include excluding dotfiles / hidden files by default which I don't imagine the crash report would generate? Looking to see if I can find an example of a previous time this actually crashed / uploaded.

Reply:
Based on the example image below from the [oss-fuzz CI instructions](https://google.github.io/oss-fuzz/getting-started/continuous-integration/) I think we should be good to leave this as-is as it seems like this is normally just a crash dump and summary:
![image](https://github.com/user-attachments/assets/f2f59771-993c-4976-ac09-1ddcdb9b54a8)